orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/servergroup/CloneServerGroupTask.groovy

```diff
@@ -86,11 +86,13 @@ class CloneServerGroupTask extends AbstractCloudProviderAwareTask implements Tas
 
     List<Map<String, Object>> descriptions = []
     // NFLX bakes images in their test account. This rigmarole is to allow the prod account access to that image.
+    Collection<String> targetRegions = operation.region ? [operation.region] :
```

Comment:
Sorry, the PR description is confusing. We typically pass regions as keys in a map field named `availabilityZones` ( `availabilityZones: [ us-east-1: [...], us-west-1: [...] }`). The preferred default is `region`.